tests/test_elbv2/test_elbv2.py

```diff
@@ -1796,7 +1793,7 @@ def test_modify_listener_http_to_https():
             Port=443,
             Protocol="HTTPS",
             SslPolicy="ELBSecurityPolicy-TLS-1-2-2017-01",
-            Certificates=[{"CertificateArn": google_arn, "IsDefault": False}],
+            Certificates=[],
```

Comment:
Would you mind adding assertions to this check? I know they weren't there in the first place, but it would be nice to have.
```
with pytest.raises() as ex:
    ...
err = ex.value.response["Error"]
err["Code"].should.equal(..)
err["Message"].should.equal(..)
```

Reply:
Absolutely, makes sense to add it: a9df01f